sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Metrics/ExecutableLinesWalker.cs

```diff
@@ -29,15 +32,23 @@ public class ExecutableLinesWalker : CSharpSyntaxWalker
     {
         private readonly HashSet<int> executableLineNumbers = new HashSet<int>();
 
+        private static readonly string[] ExcludeFromCoverageAttributes =

```

Comment:
You seem to have ignore my comment saying to use `KnownType` instead of string. Could you explain why?